CMakeLists.txt

```diff
@@ -340,6 +345,8 @@ set(GLOBAL_HEADERS
     ${CMAKE_CURRENT_BINARY_DIR}/config.h
     )
 
+set(PROJECT_NAME ${PROJECT_NAME}-qt${QT_VERSION_MAJOR})
```

Comment:
Maybe we should have another separate variable instead of modifying default cmake project_name var?

Reply:
Yeah about that one, that's what I could do with my CMake knowledge, if there's a better alternative that'll be great  :+1: